src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/KubernetesDeclarativeAgent.java

```diff
@@ -271,6 +282,9 @@ public String getSupplementalGroups() {
         if (yamlMergeStrategy != null) {
             argMap.put("yamlMergeStrategy", yamlMergeStrategy);
         }
+	if (workspaceVolume != null) {
+            argMap.put("workspaceVolume", workspaceVolume);
+	}
```

Comment:
Indent with spaces
```suggestion
        if (workspaceVolume != null) {
            argMap.put("workspaceVolume", workspaceVolume);
        }
```

Reply:
Sorry about that, poorly configured editor. Cleaned it up and committed.